library/src/main/java/com/github/paolorotolo/appintro/AppIntroBase.java

```diff
@@ -389,21 +410,40 @@ public void addSlide(@NonNull Fragment fragment) {
      * @param progressButtonEnabled Set true to display. False to hide.
      */
     public void setProgressButtonEnabled(boolean progressButtonEnabled) {
+
         this.progressButtonEnabled = progressButtonEnabled;
         if (progressButtonEnabled) {
+
             if (pager.getCurrentItem() == slidesNumber - 1) {
                 setButtonState(nextButton, false);
                 setButtonState(doneButton, true);
-                setButtonState(skipButton, false);
+                if (isWizardMode){
+                    setButtonState(backButton, showBackButtonWithDone);
+                }else {
+                    setButtonState(skipButton, false);
+                }
+
             } else {
                 setButtonState(nextButton, true);
                 setButtonState(doneButton, false);
-                setButtonState(skipButton, skipButtonEnabled ? true : false);
+                if (isWizardMode){
+                    if (pager.getCurrentItem()==0){
+                        setButtonState(backButton, false);
+                    }else {
+                        setButtonState(backButton, isWizardMode ? true : false);
+                    }
+                }else {
+                    //setButtonState(skipButton, skipButtonEnabled ? true : false);
```

Comment:
Does `skipButtonEnabled` still function as per usual for those not making use of Wizard Mode?